bundles/org.openhab.binding.ipcamera/README.md

```diff
@@ -0,0 +1,1312 @@
+2. Use any web browser to fetch this URL https://x.x.x.x/cgi-bin/CGIProxy.fcgi?cmd=getMotionDetectConfig1&usr=xxxxx&pwd=xxxxx
+3. Use the information returned by the above url to create the override settings.
+
+An example for a Foscam C2 is...
+
+```
+/cgi-bin/CGIProxy.fcgi?cmd=setMotionDetectConfig1&isEnable=1&snapInterval=1&schedule0=281474976710655&schedule1=281474976710655&schedule2=281474976710655&schedule3=281474976710655&schedule4=281474976710655&schedule5=281474976710655&schedule6=281474976710655&x1=0&y1=0&width1=10000&height1=10000&sensitivity1=1&valid1=1&linkage=6&usr=xxxxx&pwd=xxxxx
+```
+
+Another example is:
+
+```
+/cgi-bin/CGIProxy.fcgi?cmd=setMotionDetectConfig&isEnable=1&linkage=0001&sensitivity=1&triggerInterval=15&schedule0=281474976710655&schedule1=281474976710655&schedule2=281474976710655&schedule3=281474976710655&schedule4=281474976710655&schedule5=281474976710655&schedule6=281474976710655&area0=1023&area1=1023&area2=1023&area3=1023&area4=1023&area5=1023&area6=1023&area7=1023&area7=1023&area8=1023&area9=1023&usr=username&pwd=password
+```
+
+
+### Instar
+
++ For mjpeg to work you need to set the first sub-stream to be mjpeg format for the default settings to work, otherwise you can override the default with STREAM_URL_OVERRIDE with a valid url for mjpeg streams.
++ Be sure to update to the latest firmware for your camera as Instar have made a lot of improvements recently, including adding MQTT features (MQTT is not needed for this binding to work).
+
+
+## Installing the binding and discovery
+
+Installing the binding before it is merged into the project is as simple as first stopping Openhab from running, then drop all the JAR files from inside the ZIP, into the ``addons`` folder.
+Ignore any errors until multiple restarts to Openhab are made, this is to allow the cache to be created.
+
+Auto discovery can be used for ONVIF cameras, and you can either use the UI or use textual configuration which is covered below in more detail.
+
+If you use auto discovery or manually add a camera with PaperUI, it may be required to delete the camera and re-add it for the Openhab DB to be refreshed with the correct data if the binding changes greatly between versions. 
+With auto discovery the camera when found a second time should get the same name/uniqueID due to using the IP address for the unique ID by default.
+
+
+## Supported Things
+
+If using openHAB's textual configuration or when needing to setup HABPANEL/sitemaps, you are going to need to know what your camera is as a "thing type". 
+These are listed in CAPS below. Example: The thing type for a generic onvif camera is "ONVIF".
+
+| Thing Type ID | Description |
+|-|-|
+| `AMCREST` | Use for all Amcrest Cameras that do not work as a Dahua thing. This uses an older polling method for alarm detection which is not as efficient as the newer method used in Dahua. Amcrest are made by Dahua and hence their cameras can be setup as a Dahua thing. |
```

Comment:
Thing Type IDs should be camel case.

Reply:
Acknowledged. So should I change this and create breaking changes for an unknown amount of users?

Reply:
Done.